playbook.yml

```diff
@@ -115,7 +115,7 @@
       - { name: pubsub-light, version: "1.13" }
       - { name: run-condition, version: "1.2" }
       - { name: saml, version: "1.1.3" }
-      - { name: sauce-ondemand, version: "1.186" }
+      - { name: sauce-ondemand, version: "1.184" }
```

Comment:
also needs a change in cnp-jenkins-docker, and a comment with the bug and known broken versions I think

so we know to leave it in upgrades